server/methods/core/orders.js

```diff
@@ -213,6 +213,63 @@ export const methods = {
     });
   },
 
+
+  /**
+   * orders/cancelOrder
+   *
+   * @summary Start the cancel order process
+   * @param {Object} order - order object
+   * @param {Boolean} returnToStock - condition to return product to stock
+   * @return {Object} ret
+   */
+  "orders/cancelOrder": function (order, returnToStock) {
+    check(order, Object);
+    check(returnToStock, Boolean);
+
+    if (!Reaction.hasPermission("orders")) {
+      throw new Meteor.Error(403, "Access Denied");
+    }
+
+    if (!returnToStock) ordersInventoryAdjust(order._id);
+
+    let paymentMethod = orderCreditMethod(order).paymentMethod;
+    paymentMethod = Object.assign(paymentMethod, { amount: Number(paymentMethod.amount) });
+    const invoiceTotal = order.billing[0].invoice.total;
+    const shipment = order.shipping[0];
+    const itemIds = shipment.items.map((item) => {
+      return item._id;
+    });
+
+    // refund payment to customer
+    Meteor.call("orders/refunds/create", order._id, paymentMethod, Number(invoiceTotal), err => {
+      if (err) Logger.warn(err);
+    });
+
+    // send notification to user
+    const prefix = Reaction.getShopPrefix();
+    const url = `${prefix}/notifications`;
+    const sms = true;
+    Meteor.call("notification/send", order.userId, "orderCancelled", url, sms, err => {
```

Comment:
Wrapping the `(err)` in parenthesis makes this easier to read and I would like us to stay consistent. Although if there is a good reason for using this syntax I am open.